clients/cnil/pipelines.yaml

```diff
@@ -12,10 +12,20 @@ components:    # define all the building-blocks for Pipeline
       embedding_field: emb
```

Comment:
@guillaumecherel

Reply:
To  be sure, this is not related to this partiuclar issue right ? this is something we haven't spot since the CNIL pipeline creation.

Now on this one, what you mean is that we should replace ` search_fields: text` with ` search_fields: [text, name]` ? That would be an easy bugfix straight on the master branch

Reply:
Yes, this is related to #39 and will be addressed separately